source/javascripts/lib/wirsing/utils/pager.coffee

```diff
@@ -0,0 +1,14 @@
+#= require ../module
+
+# creates a list that makes paginating a list simple
+
+wirsing.module 'utils.Pager', ->
+  class Pager
+    constructor: (@options={}) ->
+      @pageSize     = @options.perPage || 30
+      @list         = []
+      @visibleCount = @pageSize
+
+    setList: (newList) -> @list = newList
+    nextPage:    -> @visibleCount += @pageSize
+    currentList: -> @list.slice(0,@visibleCount)
```

Comment:
:smile: I'll do that. I think classes are pretty nice for situations where you need to create multiple instances of one class and maybe use inheritance and you have a lot of state.

For mostly functional logic like our filters, "Haskell"-like JavaScript feels more natural to me.